webapp/lib/charts.dart

```diff
@@ -130,25 +133,29 @@ class DailyTimeseriesLineChartView {
             ..distribution = 'linear'
             ..bounds = 'ticks'
             ..time = (new chartjs.TimeScale(unit: 'day'))
-        ],
-        yAxes: [
-          new chartjs.ChartYAxe()
-            ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
+      ],
+      yAxes: [
+        new chartjs.ChartYAxe()
+          ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
         ]),
       hover: new chartjs.ChartHoverOptions()..animationDuration = 0
     );
 
-    var chartConfig = new chartjs.ChartConfiguration(type: 'line', data: chartData, options: chartOptions);
+    var chartConfig = new chartjs.ChartConfiguration(
+        type: 'line', data: chartData, options: chartOptions);
     chart = chartjs.Chart(canvas.getContext('2d'), chartConfig);
   }
 
-  void updateChart(Map<DateTime, int> updatedCountsPerDay) {
+  void updateChart(List<Map<DateTime, int>> updatedCountsPerDayList) {
```

Comment:
since you're here, can you rename `updatedCountsPerDay*` to `updatedCountsAtTimestamp*` everywhere in this method? I accidentally forgot to change it when copying it from the other chart.

Reply:
changed the name